photography/__manifest__.py

```diff
@@ -61,6 +60,7 @@
     'author': 'Odoo S.A.',
     "cloc_exclude": [
         "data/knowledge_article.xml",
+        "data/website_view.xml",
```

Comment:
Wrong file. Would be worth adding a test checking that the files listed here exist